docs/src/reference/customizing-visualization.rst

```diff
@@ -0,0 +1,111 @@
+Customizing your Auspice visualization
+======================================
+
+Just as we can specify a build-specific analysis options in the ``builds.yaml`` file, we can also specify build-specific visualization options in this directory.
+
+Looking at the ``builds.yaml`` file, the last few lines are:
```

Comment:
I'd like to refer to this file as something other than `builds.yaml` since:

1. This is an artifact of `my_profiles/` where each profile had a `builds.yaml` file.
2. We promote usage of more descriptive names like [example-data.yaml](https://github.com/nextstrain/ncov-tutorial/blob/main/example-data.yaml).

Ideas:

- **_workflow configuration file_** (not sure, since there's so many other files that "configure" the workflow)
- **_config file_** since it's given to Snakemake's `--configfile` parameter and this is the term used in [Snakemake docs](https://snakemake.readthedocs.io/en/stable/snakefiles/configuration.html)
- Add more of an explanation on the `orientation-files` page. Maybe this file can be the only "config file" while all other files are "customization" files

Reply:
Note: it's currently referenced by description in the [example-data tutorial](https://github.com/nextstrain/ncov/blob/abef0ec9805a40b353c0547011bf332ab156470d/docs/src/tutorial/example-data.rst?plain=1#L57), and as `--configfile` in the [custom-data tutorial](https://github.com/nextstrain/ncov/blob/abef0ec9805a40b353c0547011bf332ab156470d/docs/src/tutorial/custom-data.rst?plain=1#L86).

Reply:
I don't have the context of how this fits into other language used in this doc project, but to me this is a "workflow configuration file" which I might more casually refer to as a "workflow config file" or just "config file" in the context. All the same thing to me. (In places where the filename mattered/wasn't arbitrary, I would want to use that to be specific, at least once to establish context.)

Reply:
> I'd like to refer to this file as something other than builds.yaml 

👍 How about "Just as we can specify build-specific configuration via `snakemake --configfile <your-config.yaml>`, we can..."

Reply:
I think **_config file_** should be sufficient, as long as we distinguish it from other files used to configure the workflow e.g. `auspice_config.json`

Reply:
Trying to make this more reference-able in a new glossary page ([preview](https://nextstrain--894.org.readthedocs.build/projects/ncov/en/894/reference/glossary.html), work in progress).